include/readme.md

```diff
@@ -0,0 +1,57 @@
+## Updating dependencies
+Updates should be reflected in the [createDevEnvironment guide](../projectDocs/dev/createDevEnvironment.md) and the [change log](../user_docs/en/changes.t2t).
+
+### Detours
+https://github.com/microsoft/Detours
+
+Fetch latest from master
+
+### eSpeak
+https://github.com/espeak-ng/espeak-ng
+
+[eSpeak guide](./espeak.md)
+
+### ia2
+https://github.com/LinuxA11y/IAccessible2
+
+Fetch latest from master.
+Test using [the steps here](../projectDocs/design/displayModel.md).

```

Comment:
This testing is necessary for Detours, rather than IA2. DisplayModel code relies on Detours to hook GDI text out calls.